api/envoy/extensions/wasm/v3/wasm.proto

```diff
@@ -71,13 +69,18 @@ message PluginConfig {
   // `google.protobuf.Struct` is serialized as JSON before
   // passing it to the plugin. `google.protobuf.BytesValue` and
   // `google.protobuf.StringValue` are passed directly without the wrapper.
-  google.protobuf.Any configuration = 5;
+  google.protobuf.Any configuration = 4;
+
+  // If there is a fatal error on the VM (e.g. exception, abort(), on_start or on_configure return false),
```

Comment:
Between and the ECDS PR from @kyessenov, I'm starting to wonder if we need a more generic notion of fail open/closed, enable/disable, in places like the HTTP filter pipeline. I.e. it doesn't seem we should need each extension to have to repeat the basic business logic here.

Reply:
I agree.  I'm marking these as work_in_progress and then we can update this later when we have a standard mechanism.